omod-common/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/controller/SessionController.java

```diff
@@ -43,7 +45,17 @@
 	@RequestMapping(method = RequestMethod.GET)
 	@ResponseBody
 	public Object get(WebRequest request) {
-		return new SimpleObject().add("sessionId", request.getSessionId()).add("authenticated", Context.isAuthenticated());
+		boolean authenticated = Context.isAuthenticated();
+		SimpleObject session = new SimpleObject();
+		session.add("sessionId", request.getSessionId()).add("authenticated", authenticated);
+		if (authenticated) {
+			SimpleObject authenticatedUser = new SimpleObject();
+			User user = Context.getAuthenticatedUser();
+			authenticatedUser.add("uuid", user.getUuid()).add("display",

```

Comment:
Instead of manually building a SimpleObject here, you should do:

    ConversionUtil.convertToRepresentation(Context.getAuthenticatedUser(), Representation.DEFAULT)